### DISCUSSION TITLE
can I have the 3D drawing of the box?

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#528] [Q&A] [answered]
![box](https://github.com/probonopd/MiniDexed/assets/17274454/908f8731-87ad-4592-89e8-e4685e982634)

I would like to print this box but with the small OLED screen, but the STL file is not editable (by me)
Can I have a different type of file, so I can edit it? For Fusion 360 ?

Thank you!

**Comment by probonopd** [comment]
Unfortunately no, as I don't have Fusion 360 (there is no version for Linux nor FreeBSD).

Can you send me a STEP file of your desired display cutout/mount/holder?  Just model it into a flat base plate. Also let me know where it should be placed on the top "faceplate" of the case. I will see whether I can make a variant of the case based on that. No promises but I will try.

**Reply by DutchBigEd** [reply]
Fusion 360 accepts STEP-files, so I would very much like do it myself.
I will NOT distribute this file, I promise and also I can send the edited file to you as a step-file.
Also I would like to add din midi connectors.

**Reply by probonopd** [reply, marked as answer]
[topcase.zip](https://github.com/probonopd/MiniDexed/files/12275131/topcase.zip)

**Reply by DutchBigEd** [reply]
Thanks a lot!

**Reply by DutchBigEd** [reply]
[topcase_Ed v3.zip](https://github.com/probonopd/MiniDexed/files/12329150/topcase_Ed.v3.zip)

**Comment by DutchBigEd** [comment]
![hole](https://github.com/probonopd/MiniDexed/assets/17274454/85befa02-582c-4ef8-9cf8-057c5c5b466d)

Can you tell me what the purpose is of the square hole on the left side?
Cooling maybe?
Should it stay or can it go?

**Reply by Banana71** [reply]
Isn't that where the headphone jack from the DAC sits behind?

**Reply by DutchBigEd** [reply]
😁👍🏻could be, I expected a round hole for this...

**Reply by Banana71** [reply]
quadrafonie 😂

**Reply by DutchBigEd** [reply]
🤔 must be it 😁

**Reply by probonopd** [reply]
That is to access the output of the i2s DAC board which I attached to the Ethernet/USB port with double sided tape. Since there are different variants, I made the hole a bit bigger than needed ;-)

This could (and should) be engineered more nicely.

**Reply by DutchBigEd** [reply]
✅️  check

**Reply by DutchBigEd** [reply]
![20230811_102746](https://github.com/probonopd/MiniDexed/assets/17274454/2dd20599-c787-4301-a14b-e6e7b8f36ce5)
![20230811_104313](https://github.com/probonopd/MiniDexed/assets/17274454/4df3cb1d-a268-4367-9820-5ffa900d0f16)
![20230811_105757](https://github.com/probonopd/MiniDexed/assets/17274454/50299e39-b206-4ec9-ae53-f5b48d0f7881)
![20230811_105803](https://github.com/probonopd/MiniDexed/assets/17274454/670305d2-a0bc-487a-8314-9de6529cfa72)

**Reply by probonopd** [reply]
Very neat!
